flybywiresim/a32nx
# fix: Annunciator Test Function now has no effect without DC BUS 2

Fixes #7712

## Summary of Changes
Makes the annunciator test function dependent on DC Bus 2 being powered. Otherwise it has no impact.

## Screenshots (if necessary)


## References
Komp, see issue

## Additional context

Discord username (if different from GitHub): Chaoz#3065

## Testing instructions
1. Spawn C/D
2. Moving the ANN LT switch into TEST should not have any impact.
3. Connect ground power
4. Moving the ANN LT switch into TEST should behave as expected and illuminate/show test sequences
5. Use EFB ELEC Failures to disable TR1 and TR2 to depower DC BUS 2
6. Moving the ANN LT switch into TEST should not have any impact.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
